swift/ql/test/library-tests/dataflow/taint/TaintInline.expected

```diff
@@ -0,0 +1 @@
+| nsdata.swift:110:45:110:45 | bytes | Unexpected result: tainted=108 |
```

Comment:
We're missing a `$` on the `// tainted=108` comment.

Reply:
🤦‍♂️ sorry. Fixed here https://github.com/github/codeql/pull/11411.